materialfx/src/main/java/io/github/palexdev/materialfx/skins/MFXComboBoxSkin.java

```diff
@@ -124,8 +124,11 @@ private void setListeners() {
 
         comboBox.addEventHandler(MouseEvent.MOUSE_PRESSED, event -> {
             comboBox.requestFocus();
-
-            if (event.getClickCount() >= 2) {
+            // stop icon pressed event from propagating to here
+            if(event.getTarget().equals(icon.getIcon())) {
+                return;
+            }
+            if (event.getClickCount() >= 2 && event.getClickCount()%2==0) {
```

Comment:
Can you added spaces and make it look like this:
`(event.getClickCount() >= 2 && event.getClickCount() % 2 == 0)`